src/database/data-source.ts

```diff
@@ -10,6 +10,7 @@ const dataSource = new DataSource({
   type: process.env.DB_TYPE as 'postgres',
   username: process.env.DB_USERNAME,
   password: process.env.DB_PASSWORD,
+  port: 5433,
```

Comment:
Please remove. Previous settings allows it to connect on the default port of postgres which is 5432. Better change will be:
port: process.env.DB_PORT || 5432